resources/dicts/events/death/general.json

```diff
@@ -3916,7 +3916,7 @@
             "status": [ "any" ]
         },
         "history": [{
-            "cats": ["m_c" ],
+            "cats": [ "m_c", "r_c"],
```

Comment:
The "cats" list is meant for specifying which cats can have the histories in this block assigned to them.  As r_c doesn't die in this event, I'm fairly certain that, at best, this has no effect (at worst, might break something?)